hurl/admin.hurl

```diff
@@ -0,0 +1,12 @@
+# Get the admin homepage
+GET {{host}}/admin
+[Options]
+repeat: 2 # repeat so that we confirm caching behaviour
+HTTP/2 302 # redirect via HTTP/2
+[Asserts]
+duration < 1000 # ensure that it loads sub 1s - GitHub Actions networking can be slow...
```

Comment:
Is there a way to express this as a timeout? That could save some CI time if there are accidental infinite loops or really slow servers.

Reply:
We could use the `--max-time` flag - https://hurl.dev/blog/2024/08/29/hurl-5.0.0-the-parallel-edition.html#time-units - but this is a global setting.

In this case, I was going for: _we expect this response to return sub 1s, regardless where it runs_. Anything else should be considered a test failure, since the system as a whole is not behaving as expected.

I really think that we should address this properly by running both Pipedream **and** the origin in the same context as the tests, so that the networking characteristics are controlled. I think of this as a natural consequence of https://github.com/thechangelog/pipedream/pull/3#discussion_r1867220351